components/ILIAS/Export/xml/ilias_user_3_7.dtd

```diff
@@ -50,13 +50,13 @@ Id is ILIAS ID or unique identifier for this xml file
                      %I_Encoding; >
 
 <!-- AuthMode -->
-<!ENTITY % A_Mode " type  (default | local | ldap | radius | shibboleth | cas | soap | script )  'default'">
```

Comment:
Please create a new file `ilias_user_11.dtd` where you apply this change. We must not change the old files.